EduID/Flows/CreateEduID/ViewControllers/CheckEmailViewController.swift

```diff
@@ -48,6 +48,7 @@ class CheckEmailViewController: CreateEduIDBaseViewController {
         let gmailStack = UIStackView(arrangedSubviews: [gmailImage, gmailButton])
         gmailStack.alignment = .center
         gmailStack.spacing = 8
+        gmailButton.tag = 0
```

Comment:
Don't use magic numbers, but define constants for them instead.

Reply:
sure.